### PR TITLE
Align MSBuild properties across all projects

### PR DESCRIPTION
**Pull request by joegoldman2**
- Set `TargetFrameworks` to `$(SupportedTargetFrameworks)`
- Align `LanguageVersion` to 11
- Set `ImplicitUsings` to `enable` in root `Directory.Build.props`
- Miscellaneous cleanup

### CODE REVIEW

File: Src/Fido2/Fido2.csproj [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
-    <Nullable>enable</Nullable>
     <RootNamespace>Fido2NetLib</RootNamespace>
+    <Nullable>enable</Nullable>
     <IsTrimmable>true</IsTrimmable>
-    <NoWarn>IDE0057</NoWarn>
-    <LangVersion>11</LangVersion>
+    <NoWarn>$(NoWarn);IDE0057</NoWarn>
   </PropertyGroup>
-   
-  <ItemGroup>
```
Comment (iamcarbon) [inline Src/Fido2/Fido2.csproj:11]: Did these files exist previously in the project, or was this just dead code?

**Review by iamcarbon — CHANGES_REQUESTED** [review]
May we Add <ImplicitUsings>disable</ImplicitUsings> to main Fido2.csproj for now. I prefer these to remain explicit on that project and defer a decision on whether we should enable.

File: Src/Fido2/Fido2.csproj (hunk 1 above)
Comment (iamcarbon) [reply Src/Fido2/Fido2.csproj:11]: Lost the context here. These files:

```
<None Remove="Fido2MetadataServiceRepository.cs" />
<None Remove="FileSystemMetadataRepository.cs" />
<None Remove="IMetadataRepository.cs" />
<None Remove="Metadata\ConformanceMetadataRepository.cs" />
<None Remove="StaticMetadataRepository.cs" />
 ```

**Comment by iamcarbon** [comment]
Aside from disabling implicating using on Fido2.csproj, everything else looks good to me.

File: Src/Fido2.BlazorWebAssembly/Fido2.BlazorWebAssembly.csproj [hunk 2]
```diff
@@ -1,10 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk.Razor">
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
     <TargetFramework>$(SupportedTargetFrameworks)</TargetFramework>
```
Comment (Regenhardt) [inline Src/Fido2.BlazorWebAssembly/Fido2.BlazorWebAssembly.csproj:4]: I'd like one more s here to also make this *`TargetFrameworks`

Comment (iamcarbon) [reply Src/Fido2.BlazorWebAssembly/Fido2.BlazorWebAssembly.csproj:4]: FYI: After .NET8.0 is released next month, I plan on adding .NET8.0 to the TargetFramework list. Having everything managed in directory.props should make this super simple.

**Review by aseigler — APPROVED** [review]

**Comment by joegoldman2** [comment]
@iamcarbon anything you need to review this PR?

**Review by iamcarbon — APPROVED** [review]
LGTM

**Review by abergs — APPROVED** [review]

**Pull request merged**